### PR TITLE
Clean up and upgraded maven-release-plugin

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees, specially @ndeloof and @carlossg

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,7 +76,6 @@
         <version>3.0.1</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Update in to 3.0.2 in such case

Comment (recena) [inline pom.xml]: Done

**Comment by carlossg** [comment]
:bee:

File: pom.xml [hunk 2]
```diff
@@ -26,14 +26,13 @@
     </license>
   </licenses>
 
-
```
Comment (oleg-nenashev) [inline pom.xml]: Ah, you know :)

**Comment by oleg-nenashev** [comment]
:bee:

File: pom.xml (hunk 2 above)
Comment (recena) [inline pom.xml]: The PR title is clear: Clean up. :dizzy_face:

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Pull request merged**